<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LPT2609" failures="0" tests="3" name="pk_OrangeHRM.TestNG_Annotation_TimeoutMethod" time="7.009" errors="1" timestamp="2022-06-14T17:49:52 IST" skipped="2">
  <testcase classname="pk_OrangeHRM.TestNG_Annotation_TimeoutMethod" name="AddUsers_Page" time="0.000">
    <skipped/>
  </testcase> <!-- AddUsers_Page -->
  <system-out/>
  <testcase classname="pk_OrangeHRM.TestNG_Annotation_TimeoutMethod" name="VerifyAddedUser" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyAddedUser -->
  <system-out/>
  <testcase classname="pk_OrangeHRM.TestNG_Annotation_TimeoutMethod" name="Sign_On" time="7.009">
    <error message="Method pk_OrangeHRM.TestNG_Annotation_TimeoutMethod.Sign_On() didn&#039;t finish within the time-out 7000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method pk_OrangeHRM.TestNG_Annotation_TimeoutMethod.Sign_On() didn't finish within the time-out 7000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:384)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:283)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Sign_On -->
  <system-out/>
</testsuite> <!-- pk_OrangeHRM.TestNG_Annotation_TimeoutMethod -->
